Update/onik_004.txt

```diff
@@ -3686,7 +3698,7 @@ void main()
 //　カツーーンと、木こりのようないい音が沢に響き渡った￥
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　カツーーンと、木こりのようないい音が沢に響き渡った。",
-		   NULL, "A solid 'thwack' rang through out Hinamizawa, as if the job was being done by a lumberjack.", Line_Normal);
+		   NULL, "A solid \"thwack\" rang through out Hinamizawa, as if the job was being done by a lumberjack.", Line_Normal);
```

Comment:
I think this is a matter of taste whether sound effects like that should be single or double quoted, as long as it's consistent.  Probably less work to leave as is since I imagine it's consistent this way throughout the story.  Either way is fine though.